docs/cubeviz/displaycubes.rst

```diff
@@ -27,11 +29,18 @@ also be collapsed into a spectrum using the same statistic and displayed in
 the spectrum viewer along with the spectrum resulting from collapsing all the
 data in each spectral slice.
 
+.. _cubeviz-selecting-data:
 
 Selecting Data Set
 ==================
 
-Coming soon
+The data to be displayed in each viewer can be selected and de-selected by 
+clicking on the "hammer and screwdriver" icon at the top left of each viewer. 
```

Comment:
Patrick suggested using :guilabel:\`Hammer and screwdriver\` to describe UI elements.

Reply:
Good call, will push an update soon